### ISSUE TITLE
If the package is imported but not used the app crashes when the phone state changes on ios

### ISSUE THREAD

**Issue by PoeMcMoe** [andreamainella98/phone_state#6]
I'm listening for phone state on android, but not IOS. IOS crashes on receiving a call.

**Comment by andreamainella98** [comment]
Hi @PoeMcMoe, can you give me a more information? Send me your version of flutter and version iOS you used.
If you used a iOS simulator this plugin not works.

**Comment by PoeMcMoe** [comment]
Thanks for reaching out! 
I hope I can help.

Flutter 3.0.3
iOS 15.4.1
Tested on physical device with new project. All I did was add the package to the project and call the phone.

stack:
2022-06-23 14:45:35.659913+0300 Runner[49788:4948229] Metal API Validation Enabled
2022-06-23 14:45:35.724721+0300 Runner[49788:4948229] fopen failed for data file: errno = 2 (No such file or directory)
2022-06-23 14:45:35.724758+0300 Runner[49788:4948229] Errors found! Invalidating cache...
2022-06-23 14:45:35.861796+0300 Runner[49788:4948229] Warning: Unable to create restoration in progress marker file
2022-06-23 14:45:35.913882+0300 Runner[49788:4948589] flutter: The Dart VM service is listening on http://127.0.0.1:64485/S13I4l0AR24=/
2022-06-23 14:45:36.418218+0300 Runner[49788:4948572] fopen failed for data file: errno = 2 (No such file or directory)
2022-06-23 14:45:36.418281+0300 Runner[49788:4948572] Errors found! Invalidating cache...
phone_state/PhoneStateHandler.swift:34: Fatal error: Unexpectedly found nil while unwrapping an Optional value
2022-06-23 14:47:12.843655+0300 Runner[49788:4948229] phone_state/PhoneStateHandler.swift:34: Fatal error: Unexpectedly found nil while unwrapping an Optional value

Don't have any experience developing swift, but my guess is the error happens because _eventSink is nil.
